variables.tf

```diff
@@ -166,6 +166,12 @@ variable "cache_type" {
   description = "The type of storage that will be used for the AWS CodeBuild project cache. Valid values: NO_CACHE, LOCAL, and S3.  Defaults to S3.  If cache_type is S3, it will create an S3 bucket for storing codebuild cache inside"
 }
 
+variable "cache_bucket_suffix_enabled" {
+  type        = bool
+  default     = true
```

Comment:
Can you update the default to be `false` to keep the current behavior the same for folks who upgrade?

Reply:
I was under the impression `true` was the default behavior.
https://github.com/cloudposse/terraform-aws-codebuild/blob/3d9d73ac430e475486c75f7a6615c805f6837d09/variables.tf#L22

Should I proceed to set it as `false`?

Reply:
Aha apologies -- you're correct 👍 

No need to update.